src/app/meditation/meditation.component.ts

```diff
@@ -359,6 +366,48 @@ export class MeditationComponent {
     this.appState.set('isMeditating', false);
   }
 
+  /**
+   * Determines the percentage of the reached goal of commitment.
+   */
+  reached(commitment) {
```

Comment:
Do you mean importing the method from `commitment.component.ts` or adding it to `commitment.service.ts`?